### PR TITLE
Quick Start Guide Rewrite

### PR DESCRIPTION
**Pull request by deadbytefpv**
I hope this rewrite can help a lot of newcomers to the hobby and to the ExpressLRS ecosystem. 

I can't, however, help them if they choose to be lazy and skip these guides. 

Objectives:
- To hopefully make the Quick Start section of the website a lot more Newbie friendly
- To minimize wall of texts on the Quick Start section
- To make the guides less confusing both to newcomers to the hobby and new adopters
- To provide a strong foundation for incoming Hardware Flashing/Updating Guides

Spoon.

Lots of spoon.

Spoon

### CODE REVIEW

File: docs/quick-start/transmitters/firmware-version.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+For the latest ExpressLRS firmware version, check the [Releases page in Git Hub](https://github.com/ExpressLRS/ExpressLRS/releases)
+
+## I Know the Version. What Now?
+
+=== "I have an old version"
+
+    See the [next page](../transmitters/updating.md) for the typical updating procedure or use the links below to find the updating instructions specific to your TX Module.
+
+    -links-
```
Comment (maybenikhil) [inline docs/quick-start/transmitters/firmware-version.md:149]: remove

**Review by maybenikhil — CHANGES_REQUESTED** [review]
I really like how this looks. I think in terms of organization, moving installing configurator and firmware options doesnt make a ton of sense IMO but that could have been a mistake. I would also simplify the guides for the actual TXes, especially the copypasta ones to the template style because it's significantly better.

**Comment by deadbytefpv** [comment]
> I really like how this looks. I think in terms of organization, moving installing configurator and firmware options doesnt make a ton of sense IMO but that could have been a mistake. I would also simplify the guides for the actual TXes, especially the copypasta ones to the template style because it's significantly better.

Those two pages, they aren't set in stone yet. I may have to move them back and improve their contents. Simplify.
I haven't done the individual guides yet as I wanted to see feedback on the generalized steps first (which will come from the general/typical updating guide). 
Less time will be needed to "fix" the generalized guide than fix it later on individual pages. High probability of missing something on individual pages as well.

File: docs/quick-start/transmitters/firmware-version.md (hunk 1 above)
Comment (deadbytefpv) [reply docs/quick-start/transmitters/firmware-version.md:149]: My intention for this section is to provide a "directory" for all individual hardware pages, with photo links.. 

I was also thinking of using the search bar here so users can type in their hardware, and the link to the updating guide will show up. Seems mkdocs doesn't allow the search bar somewhere else than the header though.

**Review by maybenikhil — COMMENTED** [review]
I like where this PR is directionally, once you add the links to pages & the copypaste (take your time on that) i'll do a full review of it again.

**Review by maybenikhil — APPROVED** [review]
I didn't closely review each individual page, and I feel the updating page is good, so I didn't look too closely at it either. Pushing some minor spelling/grammar changes (thanks Grammarly), but other than that I'm good to go

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**